civihr_employee_portal/views/views_export/views_mydetails_personal.inc

```diff
@@ -0,0 +1,212 @@
+<?php
+
+$view = new view();
+$view->name = 'mydetails_personal';
+$view->description = '';
+$view->tag = 'default';
+$view->base_table = 'civicrm_contact';
+$view->human_name = 'MyDetails_Personal';
+$view->core = 7;
+$view->api_version = '3.0';
+$view->disabled = FALSE; /* Edit this to true to make a default view disabled initially */
+
+/* Display: Master */
+$handler = $view->new_display('default', 'Master', 'default');
+$handler->display->display_options['css_class'] = 'my-details-view-panel';
+$handler->display->display_options['use_more_always'] = FALSE;
+$handler->display->display_options['access']['type'] = 'perm';
+$handler->display->display_options['access']['perm'] = 'view my details';
+$handler->display->display_options['cache']['type'] = 'none';
+$handler->display->display_options['query']['type'] = 'views_query';
+$handler->display->display_options['query']['options']['distinct'] = TRUE;
+$handler->display->display_options['exposed_form']['type'] = 'basic';
+$handler->display->display_options['pager']['type'] = 'some';
+$handler->display->display_options['pager']['options']['items_per_page'] = '1';
+$handler->display->display_options['pager']['options']['offset'] = '0';
+$handler->display->display_options['style_plugin'] = 'default';
+$handler->display->display_options['row_plugin'] = 'fields';
+$handler->display->display_options['row_options']['hide_empty'] = TRUE;
+/* Footer: Global: Text area */
+$handler->display->display_options['footer']['area']['id'] = 'area';
+$handler->display->display_options['footer']['area']['table'] = 'views';
+$handler->display->display_options['footer']['area']['field'] = 'area';
+$handler->display->display_options['footer']['area']['empty'] = TRUE;
+$handler->display->display_options['footer']['area']['content'] = '                <a href="/user/!2/edit" class="ctools-modal-civihr-custom-style  btn btn-primary">
```

Comment:
`/user/!2/edit"` 
it cant be hardcoded, and needs to be dynamic.

Reply:
This was your suggestion and now it is a token.
`!2` is the token that holds the UID of the current logged in user.